src/trackers/ASC.py

```
@@ -109,15 +113,26 @@ async def load_localized_data(self, meta):
-
-            for data_type, result_data in zip(data_types, api_results):
-                if data_type == 'main':
-                    self.main_tmdb_data = result_data
-                elif data_type == 'season':
-                    self.season_tmdb_data = result_data
-                elif data_type == 'episode':
-                    self.episode_tmdb_data = result_data
+            try:
+                api_results = await asyncio.gather(*coroutines)
+
+                for data_type, result_data in zip(data_types, api_results):
```

Ruff (0.14.3)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)